docs/observe/observing-a-service.md

```diff
@@ -0,0 +1,90 @@
+# Observing a Service with Prometheus
+
+There are mainly two systems involved in collecting and visualizing the metrics. [Prometheus](https://prometheus.io/) is used to collect the metrics from the Ballerina service and [Grafana](https://grafana.com/) can connect to Prometheus and visualize the metrics in the dashboard.
+
+To understand how you can observe Ballerina services, let’s consider a service that converts JSON to XML.
+
+![alt text](../../assets/img/prometheus-grafana.png)
+
+As inferred by the above image, we will be getting metrics using Prometheus and displaying it using Grafana.
```

Comment:
Shall we number and describe the data flow paths in the diagram for clarity?

Reply:
Will pass on feedback to the marketing team.

Reply:
Will send this in a subsequent PR.